### ISSUE TITLE
Black screen on IdeaPad 3i Chromebook Gen 6 (Blipper; Dedede)

### ISSUE THREAD

**Issue by doe1097** [MilkyDeveloper/cb-linux#52]
Hello,
I made a USB stick following the instructions in the Docs. Seems like everything worked, but when I boot from USB the screen stays black. 

Chromebook: IdeaPad 3i Chromebook Gen 6 (Blipper)
Processor: Intel Pentium Silver N6000 (Tiger Lake)

Firmware is stock. Is there any information you need to fix this?

**Comment by animeking9901** [comment]
Confirming the same issue on Galaxy Chromebook Go (sasuke) board is also dedede

**Comment by MilkyDeveloper** [comment]
I see. It must be a problem with i915, the GPU drivers. Could you tell me the output `uname -a` from ChromeOS?

**Comment by animeking9901** [comment]
Absolutely! i didnt get a email notification for some reason,heres that output on my machine

**Comment by xSPARExSTEWx** [comment]
Same issue here with an Acer 317, which has the same N4500 CPU.
